helper/src/components/portalnav.js

```diff
@@ -109,13 +109,16 @@ export default function PortalNav({ config }) {
   const { tabLabels, defaults, presets } = config
   const [pivotkey, setPivotkey] = useState(Object.keys(tabLabels)[0])
   useAITracking("PortalNav", tabLabels[pivotkey])
-
   const [urlParams, setUrlParams] = useState(new URLSearchParams(window.location.search))
   const [invalidArray, setInvalidArray] = useState(() => Object.keys(defaults).reduce((a, c) => { return { ...a, [c]: [] } }, {}))
+  useAITracking("PageNav", urlParams.get('preset') || 'defaultOps')
```

Comment:
I'd like to see `useAITracking` here trigger on the `selected` state rarther than use the url.

so `useAITracking('PageNav', selected.preset)`